src/App.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
```suggestion
import  {PureComponent} from 'react';
```
Разве так не было бы немного красивее?